packages/mobile/src/screens/upload-screen/processTrackFile.ts

```diff
@@ -0,0 +1,25 @@
+import { newTrackMetadata } from '@audius/common'
+import type { DocumentPickerResponse } from 'react-native-document-picker'
+
+import type { UploadTrack } from './types'
+
+const ALLOWED_MAX_AUDIO_SIZE_BYTES = 250 * 1000 * 1000
```

Comment:
Is this a constraint on desktop too?

Reply:
yes it is, going to move a lot of common utilities over, including extracting photo and other things from the selected image